src/views/blogPage1.vue

```diff
@@ -0,0 +1,216 @@
+<script setup>
+
+
+
+
+</script>
+
+
+<template>
+  <div>
+    <h2>Catalogo de carros<span><theButton >Next</theButton></span></h2>
+    <ul>
+      <li>
```

Comment:
use um v-for pra fazer isso, transforma os valores que vc precisa em uma variavel e utilize um v-for para printar quantas vezes quiser ao invéz de colocar o mesmo codigo N vezes.